liiatools_pipeline/sensors/location_schedule.py

```diff
@@ -226,7 +247,9 @@ def reports_schedule(context):
         context.log.debug(f"Config used: {clean_config}")
 
         if previous_matching_run_id is None:
-            context.log.info("Differences found, executing run")
+            context.log.info(
+                f"Differences found with {clean_config.dataset} in {clean_config.dataset_folder}, executing run"
```

Comment:
`clean_config.dataset_folder` will always return None here so probably not very useful. If you're after the actual folder probably best to use `{folder_location} {dataset}` (as used above)